### ISSUE TITLE
use "%" as the trigger of LaTeX comment in {markdown} environment

### ISSUE THREAD

**Issue by yfzhao20** [Witiko/markdown#89]
Hi! I have used `markdown` package for some time and think it is a WONDERFUL package. Recently I updated `markdown` package to `2.10.0` and then tested some new features.  However I found `%` comment failed, even `hybrid` turned on,  like that:

``` tex
\documentclass{article}
\usepackage[hybrid]{markdown}
\begin{document}
\begin{markdown}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% Here is the comment here is hte comment 

Here is not the comment
    
\end{markdown}
\end{document}
```

result:

![image](https://user-images.githubusercontent.com/80528249/128627075-1129a8bd-0ecb-4fac-9d5d-303a66e21777.png)

I still want to have some LaTeX style comment in my markdown environment. Is it possible to fix that?

Cheers!

**Comment by Witiko** [comment]
@yfzhao20 Since the conversion from markdown to TeX does not obey lines, TeX comments commonly consumed more input than intended before 2.10.0, which was unintuitive and a security issue.

Since 2.10.0, TeX comments are not recognized in the `hybrid` mode (see b21fac4) but you can enable the `texComments` option, which will strip all TeX comments before the conversion from markdown to TeX in a way that obeys newlines.

TLDR: It should be sufficient to enable `texComments`. If you only used `hybrid` for TeX comments, you can now disable it.

**Comment by yfzhao20** [comment]
Thanks! It works after I enable `texComments`.
